Symfony/CS/Fixer/Symfony/PhpdocParamsFixer.php

```diff
@@ -145,6 +162,14 @@ private function fixDocBlock($content)
         return implode($lines);
     }
 
+    /**
+     * Get all matches.
+     *
+     * @param string $line
+     * @param bool   $matchCommentOnly
+     *
+     * @return string[]
```

Comment:
It's still `string[]` there are just no elements. I'm pretty sure that doc is still valid in that case. I don't know what else we'd use?